plugins/modules/database/postgresql/postgresql_user_obj_stat_info.py

```diff
@@ -251,30 +259,22 @@ def __fill_out_info(self, result, info_key=None, schema_key=None, name_key=None)
                     self.info[info_key][elem[schema_key]][elem[name_key]][key] = val
 
             if info_key in ('tables', 'indexes'):
-                relname = elem[name_key]
                 schemaname = elem[schema_key]
-                if not self.schema:
-                    result = exec_sql(self, "SELECT pg_relation_size ('%s.%s')" % (schemaname, relname),
-                                      add_to_executed=False)
-                else:
-                    relname = '%s.%s' % (self.schema, relname)
-                    result = exec_sql(self, "SELECT pg_relation_size (%s)",
-                                      query_params=(relname,),
-                                      add_to_executed=False)
+                if self.schema:
+                    schemaname = self.schema
```

Comment:
```suggestion
                if self.schema:
                    schemaname = self.schema
```
@andytom if this PR is splitted into one for trust_input and another for refactoring, we would merge the first one now.
Otherwise, i need more time to check, probably i'll do it during next week

Reply:
Done, will add those changes to #291.

Reply:
Thanks!